### PR TITLE
Feature/listing db

### PR DESCRIPTION
**Pull request by austinmcclain859**
add listing, adds in string tags

### CODE REVIEW

File: db/alembic/__pycache__/env.cpython-311.pyc
Comment (kiwi0401) [inline db/alembic/__pycache__/env.cpython-311.pyc:1]: Delete this file

File: db/alembic/versions/__pycache__/66d61b7d9162_add_listings.cpython-311.pyc
Comment (kiwi0401) [inline db/alembic/versions/__pycache__/66d61b7d9162_add_listings.cpython-311.pyc:1]: delete this file

File: db/alembic/versions/__pycache__/9017eb6eb0fd_create_users_table.cpython-311.pyc
Comment (kiwi0401) [inline db/alembic/versions/__pycache__/9017eb6eb0fd_create_users_table.cpython-311.pyc:1]: delete this file

File: db/data/__pycache__/models.cpython-311.pyc
Comment (kiwi0401) [inline db/data/__pycache__/models.cpython-311.pyc:1]: delete this file

**Review by kiwi0401 — CHANGES_REQUESTED** [review]

**Comment by austinmcclain859** [comment]
4 pyc files removed

**Review by kiwi0401 — APPROVED** [review]
LGTM!

**Pull request merged**